setup.py

```diff
@@ -99,7 +99,7 @@
     keywords="",
     install_requires=[
         "PyJWT==2.0.*",
-        "httpx==0.15.*",
+        "httpx>=0.15.0 ,<=0.22.0",
```

Comment:
```suggestion
        "httpx>=0.15.0 ,<0.23.0",
```

Reply:
We should also freeze the version for this in `development` array above cause during dev, we will all then use the same version - not causing issues.